ci/build_docs.sh

```diff
@@ -25,7 +25,8 @@ rapids-mamba-retry install \
     --channel "${PYTHON_CHANNEL}" \
     "dask-cuda=${RAPIDS_VERSION}"
 
-export RAPIDS_DOCS_DIR="$(mktemp -d)"
+RAPIDS_DOCS_DIR="$(mktemp -d)"
+export RAPIDS_DOCS_DIR
```

Comment:
This looks very weird to me and unnecessarily verbose, why is that change recommended by shellcheck?

Reply:
It's a `bash` footgun.  
If it's a single-line, then the return code of the line is the result of `export` and it swallows the return code of the captured command.

Reply:
And when running this locally, the errors will include a link explaining the issue: https://www.shellcheck.net/wiki/SC2155

Reply:
Ah makes sense, but it's also only problematic if you're running a command, if you're assigning a static value then exporting in the same line should have no problems. Thanks both!